presto-spi/src/main/java/com/facebook/presto/spi/ConnectorTableLayout.java

```diff
@@ -121,14 +131,24 @@ public ConnectorTableLayoutHandle getHandle()
      * A collection of discrete predicates describing the data in this layout. The union of
      * these predicates is expected to be equivalent to the overall predicate returned
      * by {@link #getPredicate()}. They may be used by the engine for further optimizations.
+     * This is a way to support Hive partitions.
      */
     public Optional<DiscretePredicates> getDiscretePredicates()
     {
         return discretePredicates;
     }
 
     /**
-     * Properties describing the layout of the data (grouping/sorting) within each partition
+     * Describes the layout of the data (grouping/sorting) within a split.
+     * <p>
+     * There are the following properties:
+     * - sorting property says that data per split will be coming in a sorted order. Example: 1 2 2 3 5.
```

Comment:
This won't format correctly when rendered as Javadoc. Make these HTML lists (`<ul>`, `<li>`) and probably use code formatting for the example list of numbers.